### PR TITLE
Central Command jokes or we're doing big amount of trolling.

### PR DESCRIPTION
**Pull request by OceanFish1**
## About The Pull Request

Remember you had those 2% chance that a xeno egg can spawn in xenobio at round start? This makes so at April fools' day you have 40% chance of that, but if you go check for the egg you will get an epic fail :trollface: 

## Why It's Good For The Game

April fools' expansion fun for the whole family xenobio mains crying and shaking rn

## Changelog
:cl:
tweak: On April fools' day xenobiology have a chance to get trolled by Central Command.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/spawners/xeno_egg_delivery.dm [hunk 1]
```diff
@@ -17,3 +17,22 @@
...
+
+/obj/effect/spawner/xeno_egg_delivery_troll/Initialize(mapload)
+	..()
+	var/turf/T = get_turf(src)
+	new /obj/structure/alien/egg/burst(T)
+	new /obj/item/paper/troll(T)
+	playsound(T, 'sound/items/bikehorn.ogg', 60, 0, 0)
+
+	message_admins("\"An alien egg\" has been delivered to [ADMIN_VERBOSEJMP(T)].")
+	log_game("\"An alien egg\" has been delivered to [AREACOORD(T)]")
```
Comment (Archanial) [inline code/game/objects/effects/spawners/xeno_egg_delivery.dm]: Very funny but make it say it is a joke.

**Review by PowerfulBacon — DISMISSED** [review]
if the paper isnt inside the egg then its not a good troll

**Comment by OceanFish1** [comment]
> if the paper isnt inside the egg then its not a good troll

technically it spawns inside the egg

:trollface:

**Comment by heepox** [comment]
instead of facehugger it will spawn a stickbug that dances.

**Comment by OceanFish1** [comment]
> instead of facehugger it will spawn a stickbug that dances.

not funny

**Review by utilbird — APPROVED** [review]
Significant quantities of jesting

**Comment by OceanFish1** [comment]
@PowerfulBacon trolled :trollface:

**Comment by francinum** [comment]
Good job you didn't test this at all did you.

**Pull request merged**